pkg/utils/cgroups.go

```diff
@@ -51,8 +51,8 @@ func GetContainerCgroupDir(subsystemDir, containerID string) string {
 }
 
 // GetProcessInContainer gets the IDs of all processes in the container.
-func GetProcessInContainer(cgroupParentDir, containerID string) ([]int, error) {
-	var pids []int
+func GetProcessInContainer(cgroupParentDir, containerID string, pidsptr *[]int) ([]string, error) {
```

Comment:
We could drop this `pidsptr` if it's not used anywhere?

Reply:
Removed it.